app/models/user.rb

```diff
@@ -1,12 +1,14 @@
 class User < ApplicationRecord
-  has_many :posts, foreign_key: 'author_id'
-  has_many :comments, foreign_key: 'author_id'
-  has_many :likes, foreign_key: 'author_id'
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
```

Comment:
Leaving unused code as comments in your codebase is generally not recommended. It can lead to confusion, increase code clutter, and make code maintenance more difficult. Kindly remove all commented code from your codebase to make it look more professional. :+1: